src/content/governance/index.md

```diff
@@ -0,0 +1,159 @@
+
+### Working Groups {#working-groups}
+
+An initial EIP is unlikely to be implemented on the Ethereum Mainnet without edits or changes. Generally, EIP Champions will work with a subset of Protocol Developers to specify, implement, test, iterate, and finalize their proposal. Historically, these working groups have required several months (and sometimes years!) of work. Similarly, EIP Champions for such changes should involve relevant Application/Tooling Developers early in their efforts to gather end-user feedback and mitigate any deployment risks.
+
+### Community Consensus {#community-consensus}
+
+Unfortunately, there is no clear playbook on how to handle contentious proposals. Since they have no way to force people to adopt network upgrades, Protocol Developers will generally avoid including EIPs that are contentious amongst various stakeholder groups.
+
+As the champion of a contentious EIP, you should try and build consensus, address objections and document community support for your proposal. Given the size and diversity of the Ethereum community, there isn't a single metric (e.g. a coin vote) that can be used, and EIP Champions are expected to adapt to the circumstances of their proposal.
+
+Beyond the security of the Ethereum network, significant weight is usually placed on what Application/Tooling Developers and Application Users value, given that their using and developing on Ethereum is what makes the ecosystem attractive for other stakeholders.
+
+<Divider />
+
+## Handling disagreements {#disagreements}
+
+Having many stakeholders with different motivations and beliefs means that disagreements are not uncommon.
+
+Generally, disagreements are handled with long-form discussion to understand the root of the problem. Typically, one group concedes, or a happy medium is achieved. If one group feels strongly enough, forcing through a particular change could result in a split chain. A split chain is when some stakeholders protest implementing a protocol change resulting in different, incompatible versions of the protocol operating, from which two distinct blockchains emerge.
+
+### The DAO fork {#dao-fork}
+
+Forks are when major technical upgrades or changes need to be made to the network and change the "rules" of the protocol. [Ethereum clients](/developers/docs/nodes-and-clients/) must update their software to implement the new fork rules.
+
+The DAO fork was in response to the [2016 DAO attack](https://www.coindesk.com/understanding-dao-hack-journalists) where an insecure [DAO](/glossary/#dao) contract was drained of over 3.6 million ETH in a hack. The fork moved the funds from the faulty contract to a new contract allowing anyone who lost funds in the hack to recover them.
+
+This course of action was voted on by the Ethereum community. Any ETH holder was able to vote via a transaction on [a voting platform](http://v1.carbonvote.com/). The decision to fork reached over 85% of the votes.
```

Comment:
I don't know that there is value mentioning the coin vote in this particular document for a few reasons:
1. The turnout was *incredibly* low.  If there was a quorum requirement, it definitely wouldn't have been reached.
2. Most people didn't know the coinvote was happening.
3. The coinvote only represents ETH holders, not any of the other participants in the system.
4. I may be mis-remembering, but I don't think it was a significant determining factor in the decision making process (for many of the reasons above).

Essentially, it is a great example of how Ethereum *doesn't* do governance, and I think putting it in this document just runs contrary to the goals of educating users on how Ethereum governance works.

Reply:
I think it's valuable to keep it there, but agreed it should be listed as _one part_ of the governance process and not necessarily the main thing that led to the decision. Perhaps someone with better historical context can chime in...

Reply:
yeah, i lean towards not including the coinvote as well

Reply:
Added this for clarity. Does this solve the concerns here?

> This course of action was voted on by the Ethereum community. Any ETH holder was able to vote via a transaction on [a voting platform](http://v1.carbonvote.com/). The decision to fork reached over 85% of the votes.
>
> It's important to note that whilst the protocol did fork to revert the hack, the weight the vote carried in deciding to fork is debatable for several reasons:
>
> - The turnout to vote was incredibly low
> - Most people didn't know the vote was happening
> - The vote only represented ETH holders, not any of the other participants in the system

Reply:
> Essentially, it is a great example of how Ethereum doesn't do governance, and I think putting it in this document just runs contrary to the goals of educating users on how Ethereum governance works.

Maybe we could use the above to segway into an explanation that Ethereum governance is an ever-evolving process and mistakes were made (coinvote)